ci-operator/config/openshift/multiarch/openshift-multiarch-master__nightly-4.14.yaml

```diff
@@ -178,7 +178,8 @@ tests:
   steps:
```

Comment:
`cluster:  arm01`

https://github.com/openshift/release/pull/39587/files#diff-e59de5fde7c34647729790c703192b649b577f8fb1d52e58476b8eb0eaee1887R308

Reply:
Yeah - Prashanth taught us about the `pull mode: PreserveOriginal` problem with the image streams content being pulled by the build cluster and getting stuck with the payload that matches that architecture instead of the target one.

Reply:
thanks for keeping an eye on this though ;)